network-api/networkapi/templates/pages/buyersguide/catalog.html

```diff
@@ -26,6 +26,15 @@
       </div>
     </div>
   </div>
+  <div class="tw-container">
+    <div class="tw-row">
+      <div class="tw-px-4 tw-py-5 large:tw-py-7 tw-w-full">
+        <h1 class="tw-font-zilla tw-font-light tw-text-center tw-text-2xl large:tw-text-4xl tw-m-0">
+          {% trans "Make smart choices to protect your privacy. <br /> Search for products. Read expert reviews. Get tips and tricks." %}
+        </h1>
```

Comment:
Can you add some context why this is the `h1`? This seems rather long for the title / top level heading of a page. What do you think about using "Privacy not included" as the `h1` on the home page and something like "product reviews" on the category pages? If there is nothing in the design that could act as that `h1` we could make that for screen readers only.

Reply:
changed this to a span